app/main.py

```diff
@@ -1,17 +1,30 @@
+# main.py
+
+import pytest
+
+
 def count_occurrences(phrase: str, letter: str) -> int:
-    """
-    Implement count_occurrences function:
+    """Повертає кількість появ певної літери в рядку"""
+    return phrase.lower().count(letter.lower())
+
 
-    It takes a phrase and a letter and calculates the number of times
-    the letter appears in the phrase. The function is case insensitive.
+@pytest.mark.parametrize(
+    "phrase, letter, expected_count",
+    [
+        ("hello world", "o", 2),
+        ("hello world", "x", 0),
+        ("", "a", 0),
+        ("abcd", "d", 1),
+        ("Python", "p", 1),
+    ]
+)
+def test_count_occurrences(
+    phrase: str,
+    letter: str,
+    expected_count: int
+) -> None:
+    assert count_occurrences(phrase, letter) == expected_count
 
-    count_occurrences("letter", "t") == 2
-    count_occurrences("abc", "a") == 1
-    count_occurrences("abc", "d") == 0
-    count_occurrences("ABC", "a") == 1
 
-    :param phrase: phrase to count in it
-    :param letter: letter to find occurrences of it
-    :return: count occurrences of letter in phrase
-    """
-    # write your code here
+if __name__ == "__main__":
+    pytest.main()
```

Comment:
You only need to implement the logic of the function, that is, make a counter return. Don't write tests for task, read requirements better!